src/components/FindMovie/FindMovie.tsx

```
@@ -16,42 +84,59 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              className={classNames('input', {
+                'is-danger': error,
+              })}
+              value={title}
+              onChange={e => {
+                setTitle(e.target.value);
+                setError(false);
```

Good: the input `onChange` handler clears the error state (`setError(false)`), satisfying the requirement to hide the error after the user changes the title. This makes the UX consistent with the spec.